.eslintrc

```diff
@@ -17,36 +21,22 @@
     "spread": true,
   },
   "globals": {
-    "afterEach": false,
     "assert": false,
-    "beforeEach": false,
-    "describe": false,
-    "exports": false,
-    "it": false,
-    "module": false,
-    "require": false,
```

Comment:
All these removed globals were either in `env=node` (module, exports, require), or in `env=mocha` (afterEach, beforeEach, describe, it).